lib/l10n_utils/__init__.py

```diff
@@ -164,7 +165,7 @@ class LangFilesMixin:
     add_active_locales = None
 
     def get_context_data(self, **kwargs):
-        ctx = super(LangFilesMixin, self).get_context_data(**kwargs)
+        ctx = super().get_context_data(**kwargs)
```

Comment:
Just a little left over Python 2 -> 3 cleanup. They work the same but the new way is cleaner IMHO.